config/routes.rb

```diff
@@ -1,2 +1,6 @@
 Rails.application.routes.draw do
+    get 'ingredients' => 'ingredients#index'
```

Comment:
This line is redundant. Delete this line :) We've already covered it with `resources` below.